soh/src/code/z_camera.c

```diff
@@ -1458,8 +1458,8 @@ s32 Camera_Free(Camera* camera) {
     if (RELOAD_PARAMS) {
         OLib_Vec3fDiffToVecSphGeo(&spA8, &camera->at, &camera->eye);
 
-        camera->play->camX = spA8.yaw;
-        camera->play->camY = spA8.pitch;
+        //camera->globalCtx->camX = spA8.yaw;
+        //camera->globalCtx->camY = spA8.pitch;
```

Comment:
could you switch this from commenting out to just removing these lines?

also, it'd be awesome if we had a bit of documentation as to how this fixes things. it's not clear what these lines were doing before, and why removing them resolves the issue.

just leaving a comment on this PR explaining would probably be ideal, as having a comment in code next to a lack of code would feel a little odd

Reply:
Removing these lines prevent the camera from trying and following link as he moves, fighting the free camera input.